geoportailv3/locale/.gitignore

```diff
@@ -1,2 +0,0 @@
-geoportailv3-db.pot
-geoportailv3-js.pot
```

Comment:
Yes, we should avoid adding `.gitignore` files anywhere else than the project root dir.